CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix edge cases for renderer that could result in map not rendered. ([1538](https://github.com/mapbox/mapbox-maps-android/pull/1538))
 * Fix onAnnotationDragStarted event is still fired when annotation is not draggable. ([1552](https://github.com/mapbox/mapbox-maps-android/pull/1552))
 * Fix camera flying away when pitching. ([1560](https://github.com/mapbox/mapbox-maps-android/pull/1560))
+* Fix a bug in cameraForGeometry returning incorrect camera options when pitch > 0. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
```

Comment:
Is this the only fix from dependency bump?

Reply:
Only this for public changelog entries

Reply:
Due to technical error when collecting the changelog in gl-native entry `Fix Android memory leak when destroying platform view annotation manager.` is missing, it will be added shortly, could you please update add it in our changelog as well?

Reply:
Sure